models/articles.models.js

```diff
@@ -16,4 +16,17 @@ function fetchArticleByID(article_id) {
     })
 }
 
-module.exports = fetchArticleByID;
\ No newline at end of file
+function fetchArticles() {
+    console.log('in models')
+    return db
+    .query(`SELECT comments.article_id,articles.author,articles.title,articles.topic,articles.created_at,articles.votes,articles.article_img_url, SUM(comments.votes) AS comment_count
```

Comment:
You SUM aggregate function is not the function that you want to be using. What you are actually returning as a comment_count is the total number of votes that exist. Not the number of comments that exist for each article. 


You need to use the COUNT aggregate function instead